test/trackler/problem_test.rb

```diff
@@ -82,6 +82,10 @@ def test_track_with_no_canonical_data
     assert_equal nil, problem.canonical_data_url
   end
 
+  def test_problem_with_no_metadata_yml
+    refute Trackler::Problem.new('no-metadata', FIXTURE_PATH).exists?
+  end
```

Comment:
I think you force pushed and orphaned these comments: https://github.com/exercism/trackler/commit/7bfbffe4fc1d5735db798d01ef33b08f94024b6e